### DISCUSSION TITLE
Show song percentage on F ranked replays

### DISCUSSION THREAD

**Discussion by katashisano** [ppy/osu#26427] [Ideas]
As it says in the title, it would be a cool feature to show the song progress percentage (like the one when you fail) on F ranked (failed) replays when hovering over the F label. I'm personally someone who saves a lot of failed replays and it would be a nice small QOL detail.

**Comment by mirsman47-pixel** [comment]
true, imo would be better if where scores shown if there rank F there should be something like % of map you passed

**Comment by peppy** [comment]
<img width="830" height="769" alt="osu! 2026-07-16 at 06 31 42" src="https://github.com/user-attachments/assets/91a5624d-d955-4a17-a4f4-cdd8fb159487" />

**Reply by katashisano** [reply]
> <img width="830" height="769" alt="osu! 2026-07-16 at 06 31 42" src="https://github.com/user-attachments/assets/91a5624d-d955-4a17-a4f4-cdd8fb159487" />
> 
> 

Sorry, I mean in the local scores list during song select, the picture *does not* reflect what I'm suggesting
